posts/an-introduction-to-dqm.md

```diff
@@ -0,0 +1,63 @@
+
+# Why do you need Data Quality Management Processes in Place?
+Most of a company's operations and decisions rely heavily on data, so the emphasis on data quality is even higher. Data quality refers to the assessment of information collected relative to its purpose and its ability to serve that purpose. Low-quality data costs organizations daily. An estimate by IBM calculated that the annual cost of data quality-related issues amounted to around [$3.1 trillion](https://www.ibmbigdatahub.com/infographic/four-vs-big-data).
+
+Good data quality increases the accuracy of analytics applications, which can lead to better business decision-making that boosts sales and improves internal processes. High-quality data can help expand the use of BI dashboards and analytics tools, as well. If analytics data is seen as trustworthy, business users are more likely to rely on it instead of basing decisions on their gut feelings or spreadsheets.
+
+# How do you measure Data Quality?
+There are five categories of metrics that you can use to measure data quality. These metrics can help you track the effectiveness of your data quality improvement efforts. Let's go over the categories of metrics and what they track (and, more importantly, why each is important).
+
+## Accuracy
+Accuracy tracks the number of known errors, such as those resulting from missing, incomplete, or redundant data entry. Accuracy is typically measured as a ratio of data to errors. This ratio should increase over time, indicating that the quality of your data is improving.
+
+Consider NASA's Mars Climate Orbiter. On September 23rd, 1999, [NASA's Mars Climate Orbiter](https://www.wired.com/2010/11/1110mars-climate-observer-report/) lost communication with mission control as it approached orbit around the planet. It burned up in the Martian atmosphere as it descended towards the planet's surface. An investigation revealed the two software applications which controlled the orbiter's thrusters had miscommunicated the amount of force needed to reach the right altitude. One application calculated the force in *pounds*, while the other assumed the force was in *newtons*. Nobody was harmed as a result of this mistake, other than the 125 million dollar spacecraft, but it does highlight the impact that data accuracy issues can have.
+
+## Consistency
+Consistency means ensuring that data across all systems reflect the same information and synch with each other across the enterprise. This may seem easy enough at first, but consider the case where two systems have a field named customer_id. Customer_Id means something different in each system. This is, after all, a common scenario when looking at systems across an enterprise.
```

Comment:
```suggestion
Consistency means ensuring that data across all systems reflect the same information and synch with each other across the enterprise. This may seem easy enough at first, but consider the case where two systems have a field named `customer_id`. `customer_id` means something different in each system. This is, after all, a common scenario when looking at systems across an enterprise.
```